tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1665,4 +1665,70 @@ public function theAdministratorTriesToAddTheFollowingUsersToAGroupAtOnceUsingTh
 		}
 		$this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
 	}
+
+	/**
+	 * @When user :user gets details of the group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($groupName);
+		$this->getDetailsOfGroup($user, $groupId);
+	}
+
+	/**
+	 * @When user :user gets details of non-existing group using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfNonExistingGroupUsingTheGraphApi(string $user): void {
+		// generates random group id for non-existing group
+		$groupId = WebDavHelper::generateUUIDv4();
+		$this->getDetailsOfGroup($user, $groupId);
```

Comment:
```suggestion
		$this->getDetailsOfGroup($user, WebDavHelper::generateUUIDv4());
```
same here, also do we need to `setResponse` here?